keanu-python/tests/test_appveyor_hanging.py

```diff
@@ -0,0 +1,25 @@
+import keanu as kn
+import numpy as np
+import sys
+import logging
+
+
+def test_appveyor_hanging():
+    print("test_appveyor_hanging")
+    logging.warn("test_appveyor_hanging")
```

Comment:
If we are keeping this - can you swap this for `logging.warning` - it's chucking out some annoying deprecated messages when you build